index.js

```diff
@@ -232,6 +232,7 @@ export default class QRCodeScanner extends Component {
               style={[styles.camera, this.props.cameraStyle]}
               onBarCodeRead={this._handleBarCodeRead.bind(this)}
               type={this.props.cameraType}
+              flashMode={this.props.flashMode}
```

Comment:
can we update the docs with this prop?
and if there are any default values, specifying those

Reply:
If flashMode is not set in the qrcode-scanner component the default value is OFF.

If it's set it will be defined by the RNCamera.Constant set by user:

Yes, you can update the docs:
**flashMode** 
receives: RNCamera.Constants.FlashMode.torch -> Flash On
receives: RNCamera.Constants.FlashMode.off -> Flash Off

Remember to always reactivate the qr-code-scanner to reset the props :)

If user does not set flashMode in the qr-code-scanner component:
Flash will be (always) Off

This is how it works:
this.state.isFlashOn is a boolean set by user click in a flash logo.

**flashMode={this.state.isFlashOn ? RNCamera.Constants.FlashMode.torch : RNCamera.Constants.FlashMode.off}**

See my application using this update to see how it works: 
https://play.google.com/store/apps/details?id=com.flipon.app&hl=pt_BR

Basically it uses the field that already exists in react-native-camera and import user preferences.

I have tested it on local, changing the index.js and it worked fine.

Reply:
it will be add like this:

**flashMode**
Values: RNCamera.Constants.FlashMode.off (default), RNCamera.Constants.FlashMode.on, RNCamera.Constants.FlashMode.auto or RNCamera.Constants.FlashMode.torch.

Specifies the flash mode of your camera.

RNCamera.Constants.FlashMode.off turns it off.

RNCamera.Constants.FlashMode.on means camera will use flash in all photos taken.

RNCamera.Constants.FlashMode.auto leaves your phone to decide when to use flash when taking photos, based on the lightning conditions that the camera observes.

RNCamera.Constants.FlashMode.torch turns on torch mode, meaning the flash light will be turned on all the time (even before taking photo) just like a flashlight.

Reply:
@moaazsidat, I already added to the documentation, you can see two commits. The first I added the text i send in the last comment, the second was because I have to do some edit it to keep up with the style of the rest documentation :P